<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtmHomPage</class>
 <widget class="QDialog" name="CtmHomPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_top">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Customer Home Page</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>700</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="BeginDateEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>12</month>
     <day>22</day>
    </datetime>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>12</month>
     <day>22</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="EndDateEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>12</month>
     <day>23</day>
    </datetime>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>12</month>
     <day>23</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="BeginDateLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EndDateLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>End Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="PlaceLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TypeLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PerInfoBtn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Personal Info</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="PlaceLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Hotel Place</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TypeLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Room Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchRoomBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>630</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>search room</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoutBtn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>100</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CommentSortBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>630</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort(Comment)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PriceSortBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>630</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort(Price)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubmitBtn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>630</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CheckMyOrderBtn">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>630</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Check My Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="d_TodayLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="KnowHtlInfoBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>630</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>KnowHtlInfo</string>
   </property>
  </widget>
  <widget class="QLabel" name="TodayLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Today is</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
